### PR TITLE
Axelar Bridge

### PR DESCRIPTION
**Pull request by oeggert**

### CODE REVIEW

**Comment by oeggert** [comment]
Refer to the new PR #82

File: docs/axelar/intro-to-axelar.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Axelar Integration
+
+The Axelar-XRPL integration enables messages to pass between the XRPL and other Axelar-supported blockchains. This integration enables the transfer of XRP and XRPL-issued tokens to other chains and back; this also works for briding non-XRPL tokens from their native chains to XRPL.
```
Comment (ajitkulk) [inline docs/axelar/intro-to-axelar.md:3]: suggest changing to "XRPL and other blockchains that are connected through the Axelar network"

Comment (ajitkulk) [reply docs/axelar/intro-to-axelar.md:3]: 1. Suggest a change to "transfer of XRPL- issued tokens from the XRPL to other chains and the transfer of tokens on other chains (such as ERC-20 tokens on EVM compatible chains) to the XRPL"
2. The asset XRP won't go through the Axelar bridge.

Comment (ajitkulk) [reply docs/axelar/intro-to-axelar.md:3]: 1. Suggest a change to "transfer of XRPL- issued tokens from the XRPL to other chains and the transfer of tokens on other chains (such as ERC-20 tokens on EVM compatible chains) to the XRPL"
2. The asset XRP won't go through the Axelar bridge. 
3. typo in 'briding' -> change to 'bridging'

File: docs/axelar/intro-to-axelar.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+In addition to cross-chain features, the Axelar Amplifier network utilizes tickets to enable multiple in-flight transactions to improve throughput.
+
+At a high-level, there are two main components that enable the Axelar-XRPL integration. On the XRPL-side, a multisig account is created; this account handles all incoming and outgoing transactions to the Axelar network by:
+
+- Holding and releasing assets as native XRP assets are moved off and on-chain.
```
Comment (ajitkulk) [inline docs/axelar/intro-to-axelar.md:9]: unclear suggest changing to
 For outbound transactions from XRPL -> Axelar -> another chain

-  locking XRPL-issued assets and asking the Axelar network to issue a wrapped asset on the Axelar network to send to the other chain
- burning non-XRPL wrapped asset on the XRPL and asking the Axelar network to unlock the non-XRP asset on the other chain

 For inbound transactions from Another chain  -> Axelar -> XRPL

- minting a non-XRPL wrapped asset on the XRPL after the original non-XRP asset has been locked on the other chain
- Unlocking the XRP-issued asset after the wrapped version of the asset has been burnt on the other chain and the Axelar network.

File: docs/axelar/intro-to-axelar.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+- Issuing and burning wrapped assets as non-native XRP assets are bridged to XRPL.
+
+The multisig account is ultimately controlled by the Axelar Network validators, which serve as signers. The XRPL multisig account signers and their weights are kept in sync with the state of the validators on the Axelar network. The Axelar validators' are responsible for verifying transactions on multiple chains, minting and burning as assets are bridged.
```
Comment (ajitkulk) [inline docs/axelar/intro-to-axelar.md:12]: More details here
The XRPL - Axelar connection is secured by an XRPL account owned by Axelar validators. The top 32 validators, chosen by those staking the most AXL on the Axelar PoS network, make up the signer list for the Axelar XRPL account. These 32 validators control the multi-sig account and must agree before sending transactions across the Axelar network. Interested developers can find more details on the security of the Axelar network [here](https://axelar.network/blog/security-at-axelar-core).

File: docs/axelar/intro-to-axelar.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+**Note:** Future enhancements are planned, such as General Message Passing (GMP), which would enable transactions on XRPL to trigger smart contract calls on other chains.
```
Comment (ajitkulk) [inline docs/axelar/intro-to-axelar.md:14]: This is being done now and is not a future enhancement. The other page, "Call a smart contract," details that.

File: docs/axelar/intro-to-axelar.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+8. If the poll ends successfully, the created tickets are available to be assigned to new `Payment` transactions.
+
+
+### Ticketing Logic
+
+A set of XRPL multisig tickets are always available: the tickets that have been created with `TicketCreate` but that are not known to have been consumed by transactions that made it to the XRPL ledger. If the number of available tickets falls below a configurable threshold, a new `TicketCreate` transaction is constructed and broadcasted. New transactions are assigned available tickets until they are all exhausted.
+
+![Ticketing Logic 1](../img/axelar-ticket-1.png)
+
+At that point, ticket assignment will wrap around to the first available tickets (similarl to “round robin” load balancing algorithms), leading to multiple tickets being assigned to the same transaction.
```
Comment (ajitkulk) [inline docs/axelar/intro-to-axelar.md:83]: typo on 'similarly'

File: index.mdx [hunk 6]
```diff
@@ -38,6 +38,37 @@ export default LandingContainer;
   <Header2>Explore open source projects currently in development.</Header2>
 </Jumbotron>
 
+<Box p={{ _: "8px", sm: "12px", md: "16px", lg: "24px" }}>
+  <FlexSection justifyContent="space-around" flexWrap="wrap">
+    <WideTile 
+      style={{ height: "284px"}}
+      to="docs/axelar/intro-to-axelar.md"
+      header="Axelar Bridge"
```
Comment (ajitkulk) [inline index.mdx:46]: Can we call this Axelar Network integration?

File: index.mdx [hunk 7]
```diff
@@ -38,6 +38,37 @@ export default LandingContainer;
...
+      color="#141A1F"
+      textAlign="left"
+      disableArrow
+    >
+      <TileText>
+        The Axelar Network is a Web3 interoperability platform.
```
Comment (ajitkulk) [inline index.mdx:52]: where can I find this text in the preview?

File: index.mdx [hunk 8]
```diff
@@ -38,6 +38,37 @@ export default LandingContainer;
   <Header2>Explore open source projects currently in development.</Header2>
 </Jumbotron>
 
+<Box p={{ _: "8px", sm: "12px", md: "16px", lg: "24px" }}>
+  <FlexSection justifyContent="space-around" flexWrap="wrap">
+    <WideTile 
+      style={{ height: "284px"}}
+      to="docs/axelar/intro-to-axelar.md"
```
Comment (ajitkulk) [inline index.mdx:45]: In the future, we will have another dropdown within the Axelar section for "Axelar - XRPL EVM Sidechain Integration". No action is needed now.

File: docs/axelar/intro-to-axelar.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+The [XRPL multi-signing account](https://xrpl.org/docs/tutorials/how-tos/manage-account-settings/set-up-multi-signing) handles bridging by releasing/issuing assets, or issuing/burning assets. On EVM and Cosmos chains, the equivalent is a Gateway smart contract.
+
+The Axelar Networks is enabled by four smart contracts:
+- **Gateway:** A smart contract that handles incoming and outgoing messages for a particular chain.
+- **Voting Verifier:** A smart contract that verifies the authenticity and status of transactions on the XRPL multisig account by counting votes submitted by Axelar validators.
+- **Multisig Prover:** Constructs and serializes XRPL multisig transactions to be transmitted to XRPL. Calls the “Multisig” contract on Axelar to initiate a signing session.
+- **Multisig:** Handles collection of signatures and keeps record of the weight of each signer.
+
+### Off-chain:
+– **relayer:** A permissionless off-chain process that informs the Axelar network of new transactions to be bridged from a source chain, or submits transactions prepared by the Amplifier network to a destination chain. The relayer consists of these sub-components:
```
Comment (ajitkulk) [inline docs/axelar/intro-to-axelar.md:30]: - I would suggest that this is a bullet list just like in the on-chain section above. 
- change relayer to 'Relayer'

File: docs/axelar/bridge-tokens-axelar.md [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+    ```sh
+    AXELAR_GATEWAY=0xAABdd46ba1B3147d0Cf6aCc9605a74fE8668fC74
+    XRPL_DESTINATION= # your XRPL recipient address
+    cast send $AXELAR_GATEWAY "sendToken(string destinationChain, string destinationAddress, string symbol, uint256 amount)" "xrpl" $XRPL_DESTINATION "WETH" $(cast to-wei 0.1) --private-key $PRIVATE_KEY --rpc-url $SEPOLIA_RPC_URL
+    ```
+
+5. The destination address should receive the bridged ETH in a few minutes. You can check for a successful transaction on the [XRPL Testnet explorer](https://testnet.xrpl.org/).
+
+
+### Bridge ETH from XRPL to Ethereum
```
Comment (ajitkulk) [inline docs/axelar/bridge-tokens-axelar.md:71]: Change to "Bridge wrapped ETH back to Ethereum Sepolia"
Note that there is no native ETH on the XRPL. In this section, we are just bridging back the wrapped ETH.

File: docs/axelar/bridge-tokens-axelar.md [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+                MemoData: "0000000000000000000000000000000000000000000000000000000000000000", // bytes32(0) indicates pure token transfer, without GMP
+                MemoType: "7061796C6F61645F68617368", // hex("payload_hash")
+            },
+        },
+    ],
+    ...
+}
+```
+
+2. Within a few minutes, the `destination_address` should receive the bridged XRP on Ethereum Sepolia, as axlXRP. You can check for the transaction on the [Ethereum Sepolia explorer](https://sepolia.etherscan.io/).
```
Comment (ajitkulk) [inline docs/axelar/bridge-tokens-axelar.md:146]: please specify that axlXRP is Axelar wrapped XRP

File: docs/axelar/intro-to-axelar.md (hunk 1 above)
Comment (ajitkulk) [inline docs/axelar/intro-to-axelar.md:3]: Here is a diagram that may be helpful. Let me know if you need it changed in any way to make it easier to understand
XRPL EVM Axelar Topology2

File: docs/axelar/bridge-tokens-axelar.md [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+- Funded wallets on both chains.
+  - **ETH Sepolia Faucet:** [alchemy.com/faucets/ethereum-sepolia](https://www.alchemy.com/faucets/ethereum-sepolia)
+  - **XRPL Testnet Faucet:** [faucet.tequ.dev](https://faucet.tequ.dev/)
+- [XRPL JS Library](https://js.xrpl.org/)
+
+
+## Steps
+
+### Bridge ETH from Ethereum to XRPL
+
```
Comment (ajitkulk) [inline docs/axelar/bridge-tokens-axelar.md:21]: How about this diagram to explain tehe flow at a high level?
XRPL Axelar WETH

File: docs/axelar/bridge-tokens-axelar.md [hunk 13]
```diff
@@ -0,0 +1,172 @@
...
+    ],
+    ...
+}
+```
+
+2. Within a few minutes, the `destination_address` should receive the bridged ETH on Ethereum `Sepolia`, as WETH. You can check for the transaction on the [Ethereum Sepolia explorer](https://sepolia.etherscan.io/).
+
+
+### Bridge XRP from XRPL to Ethereum
+
```
Comment (ajitkulk) [inline docs/axelar/bridge-tokens-axelar.md:113]: How about this diagram to explain the flow at a high level?
XRPL Axelar XRP axlXRP

File: index.mdx (hunk 7 above)
Comment (oeggert) [reply index.mdx:52]: This is available on the main landing pages tiles, viewable with this preview: https://preview.redoc.ly/opensource-ripple/axelar-preview/

**Pull request closed without merging**